contributing/ci-bio-formats.txt

```diff
@@ -14,6 +14,10 @@ All jobs are listed under the :jenkinsview:`Bio-Formats` view tab of Jenkins.
 		* :term:`BIOFORMATS-5.1-latest`
 		*
 
+	-	* Generates the latest generated-sources
+		* :term:`BIOFORMATS-5.1-latest-generated-sources`
```

Comment:
@rleigh-dundee: correct, this job commits the baseline for comparison used in the merge job (and potentially for other jobs like BIOFORMATS-5.1-breaking-generated-sources). I will add a sentence to that effect to the job description if that makes sense.